modules/pubmaticServerBidAdapter.js

```diff
@@ -234,32 +225,36 @@ export const spec = {
   interpretResponse: (response, request) => {
     const bidResponses = [];
     try {
-      if (response.body && response.body.seatbid && response.body.seatbid[0] && response.body.seatbid[0].bid) {
-        response.body.seatbid[0].bid.forEach(bid => {
-          if (bid.id !== null && bid.ext.summary) {
-            bid.ext.summary.forEach((summary, index) => {
-              if (summary.bidder) {
-                const firstSummary = index === 0;
-                const newBid = {
-                  requestId: bid.impid,
-                  bidderCode: BIDDER_CODE,
-                  originalBidder: summary.bidder,
-                  pubmaticServerErrorCode: summary.errorCode,
-                  cpm: (parseFloat(summary.bid) || 0).toFixed(2),
-                  width: summary.width,
-                  height: summary.height,
-                  creativeId: firstSummary ? (bid.crid || bid.id) : bid.id,
-                  dealId: firstSummary ? (bid.dealid || UNDEFINED) : UNDEFINED,
-                  currency: CURRENCY,
-                  netRevenue: true,
-                  ttl: 300,
-                  referrer: utils.getTopWindowUrl(),
-                  ad: firstSummary ? bid.adm : ''
-                };
-                bidResponses.push(newBid);
-              }
-            });
-          }
+      if (response.body && response.body.seatbid) {
+        // Supporting multiple bid responses for same adSize
+        response.body.seatbid.forEach(seatbidder => {
+          seatbidder.bid &&
+          seatbidder.bid.forEach(bid => {
+            if (bid.id !== null && bid.ext.summary) {
+              bid.ext.summary.forEach((summary, index) => {
+                if (summary.bidder) {
+                  const firstSummary = index === 0;
+                  const newBid = {
+                    requestId: bid.impid,
+                    bidderCode: BIDDER_CODE,
+                    originalBidder: summary.bidder,
+                    pubmaticServerErrorCode: summary.errorCode,
+                    cpm: (parseFloat(summary.bid) || 0).toFixed(2),
+                    width: summary.width,
+                    height: summary.height,
+                    creativeId: firstSummary ? (bid.crid || bid.id) : bid.id,
+                    dealId: firstSummary ? (bid.dealid || UNDEFINED) : UNDEFINED,
+                    currency: CURRENCY,
+                    netRevenue: true,
+                    ttl: 300,
+                    referrer: utils.getTopWindowUrl(),
```

Comment:
as function will return same value for a given loop, use variable than always executing function